src/util/numeric.h

```diff
@@ -144,36 +125,17 @@ inline bool isInArea(v3s16 p, v3s16 d)
 	);
 }
 
-#define rangelim(d, min, max) ((d) < (min) ? (min) : ((d)>(max)?(max):(d)))
-#define myfloor(x) ((x) > 0.0 ? (int)(x) : (int)(x) - 1)
-
 inline v3s16 arealim(v3s16 p, s16 d)
 {
-	if(p.X < 0)
```

Comment:
From the [kernel code style](https://www.kernel.org/doc/Documentation/CodingStyle)
```
Don't put multiple statements on a single line unless you have
something to hide:

	if (condition) do_this;
	  do_something_everytime;
```